suru-icons: Add NFC icons

fredldotme opened the pull request:
Adds active and inactive SVGs for NFC.
Also adds a symlink from 'nfc.svg' to the active SVG.

Thanks to CiberSheep (https://github.com/cibersheep) for the icons.

dobey reviewed (changes requested):
Should we not instead use the official NFC Forum logo, or the three horizontally oriented radio wave icon to indicate NFC instead @cibersheep ?

cibersheep commented:
> Should we not instead use the official NFC Forum logo, or the three horizontally oriented radio wave icon to indicate NFC instead @cibersheep ?

We thought about it and discuss it. It is not clear on what license they are published (I think at least under fair use) but we couldn't find info. We decide it to create a free one. If we can adapt one of official, then we can replace this.

fredldotme committed:
suru-icons: Add NFC icons

Adds active, disabled and idle (default) SVGs for NFC.

Thanks to Mateo Salta (https://github.com/mateosalta) and
CiberSheep (https://github.com/cibersheep) for the icons.

fredldotme commented:
New icons as decided in the UX & Graphics channel are force-pushed.
